src/app/src/app.actions.js

```diff
@@ -1,11 +1,39 @@
 import { createAction } from 'redux-act';
 
+import {
+    parseRiverGaugeApiData,
+    parseRiverGaugeCsvData,
+    makeRiverGaugeRequest,
+} from './sensorUtils';
+
 export const hideIntro = createAction('Hide intro screen');
 export const selectSensor = createAction('Select sensor');
 export const deselectSensor = createAction('Deselect sensor');
 export const showSensorModal = createAction('Show sensor modal');
 export const hideSensorModal = createAction('Hide sensor modal');
-export const setSensorData = createAction('Set sensor data');
 export const startPollingSensor = createAction('Start polling sensor for data');
 export const failPollingSensor = createAction('Failed polling sensor for data');
-export const completePollingSensor = createAction('Complete polling sensor for data');
+export const completePollingSensor = createAction(
+    'Complete polling sensor for data'
+);
+
+export function pollSensor(sensor) {
+    return async () => {
+        try {
+            startPollingSensor();
```

Comment:
Does this need to be wrapped in `dispatch` to be sent to Redux?

Reply:
No, because it is wired up to the store through `redux-act`. I didn't realize that would percolate to the actions file itself, but it does!